### PR TITLE
Euclidean Algo added

### LINKED ISSUE #2905: Euclidean Algorithm
- [x] C - @simrann20
- [x] C++
- [x] C#
- [x] CoffeeScript
- [x] Clojure
- [ ] Dart - @disha2sinha
- [ ] Go
- [x] Java
- [x] JS - @Satyam3976
- [x] Kotlin
- [x] Php
- [x] Python
- [x] Readme
- [x] Ruby - @Tahanima
- [ ] Rust - @mightomi
- [x] Scala 
- [ ] TypeScript - @shivamgupta57121

### PR DESCRIPTION
**Pull request by Satyam3976**
Fixes #2905 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Euclidean_Algorithm.js added
- 

#### Languages Used:
- JavaScript
- 

#### Files Added:
- Euclidean_Algorithm.js
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Euclidean_Algorithm/Euclidean_Algorithm.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		return b;
+	}
+	if(b == 0)
+	{
+		return a;
+	}
+	return gcd(b, a % b);
+}
+
+var result = gcd(9, 6);
```
Comment (atarax665) [inline Euclidean_Algorithm/Euclidean_Algorithm.js:18]: Take input dynamically.

File: Euclidean_Algorithm/Euclidean_Algorithm.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+// Euclidean Algorithm to find GCD of two numbers
+
+// Function to return gcd of a and b
+function gcd(a, b)
+{
+	// base cases
+	if(a == 0)
```
Comment (atarax665) [inline Euclidean_Algorithm/Euclidean_Algorithm.js:7]: The case if(a == 0) is of no use. Remove this

**Comment by atarax665** [comment]
Fill the description box. Mention the issue you are fixing.

**Comment by atarax665** [comment]
Looks fine now. Please squash your commits.

**Comment by Satyam3976** [comment]
> Looks fine now. Please squash your commits.

Done @atarax665

**Pull request merged**